test/RevenuePool-test.js

```diff
@@ -239,6 +239,42 @@ contract("RevenuePool", (accounts) => {
         prepaidCardManager.address
       );
     });
+
+    it("has a sane upper bound for merchantRegistrationFeeInSPEND", async () => {
+      await revenuePool
+        .setup(
+          exchange.address,
+          merchantManager.address,
+          actionDispatcher.address,
+          prepaidCardManager.address,
+          merchantFeeReceiver,
+          10000000,
```

Comment:
Nitpick: when passing numbers into solidity calls, it's best practice to pass as a string, i.e. `10000000` (or use [BigNumber](https://docs.ethers.io/v5/api/utils/bignumber/)). This is because javascript's Number type is only precise up to 2^53 and uint256 goes up to 2^256 - for these hardcoded low values not an issue though

Reply:
Noted, thanks!